### PR TITLE
Resuable Result Pages

### PR DESCRIPTION
**Pull request by swolbean**

### CODE REVIEW

**Comment by ClodaghMM** [comment]
Hey Bea, can you please add a screenshot of the UI you made so we can review?

**Comment by swolbean** [comment]
Screenshot (20)
Screenshot (19)

**Review by ClodaghMM — COMMENTED** [review]
Looks good. This is just the UI right? No logic has been connected yet?

**Comment by swolbean** [comment]
> Looks good. This is just the UI right? No logic has been connected yet?

Just UI, the button does take the user to the homepage but that's all the logic done

File: client/src/App.js [hunk 1]
```diff
@@ -3,17 +3,16 @@ import {BrowserRouter as Router, Route, Routes} from 'react-router-dom';
...
-import UploadImage from './views/UploadImage';
 import PICResult1 from './views/PICResult1';
 import PICResult2 from './views/PICResult2';
 import PICResult3 from './views/PICResult3';
 import PICResult4 from './views/PICResult4';
 import PICResult5 from './views/PICResult5';
 import PICResult6 from './views/PICResult6';
 import PICResult7 from './views/PICResult7';
-import Account from './views/Account'
+import Account from './views/Account';
```
Comment (JasmineBautista) [inline client/src/App.js]: ?

**Comment by JasmineBautista** [comment]
nice shell, I would merge. However we need to make a note to get these hooked up ASAP

**Comment by swolbean** [comment]
Updated pages with no white borders 

image
image

File: client/public/index.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Hamlet</title>
```
Comment (JasmineBautista) [inline client/public/index.html:27]: lol

**Review by JasmineBautista — APPROVED** [review]
yeah ok why not

**Pull request merged**